lms/static/certificates/sass/_layouts.scss

```diff
@@ -0,0 +1,324 @@
+// ------------------------------
+// // Open edX Certificates: Layouts
+
+// About: styling for generic layouts and responsive support.
+
+// #BASE
+// #ACCOMPLISHMENT
+
+// ------------------------------
+// #BASE
+// ------------------------------
+%layout-wrapper {
+    margin-bottom: spacing-vertical(base);
+    padding: 0 (gutter()*2);
+
+    @include susy-breakpoint($bp-medium, $susy) {
```

Comment:
NOTE: new susy grid-management syntax. ``susy-breakpoint`` is a mixin that combines susy with breakpoint (http://breakpoint-sass.com/) for easy mgmt of breakpoints. ``gutter()`` is another handy susy function for getting the width of your grid's gutters out of a grid layout context.

See http://www.sitepoint.com/creating-responsive-grid-system-susy-breakpoint/ and http://susydocs.oddbird.net/en/latest/.